liblangutil/ErrorReporter.cpp

```diff
@@ -39,6 +39,9 @@ ErrorReporter& ErrorReporter::operator=(ErrorReporter const& _errorReporter)
 
 void ErrorReporter::warning(string const& _description)
 {
+	if (_description == "This is a pre-release compiler version, please do not use it in production.")
```

Comment:
What is the purpose of this line?

Reply:
On CircleCI a release version is build. So on CircleCI where will be always one warning less than in on a developers machine using a pre-release build.

Reply:
It actually builds both versions on CircleCI. Maybe the syntax test framework is no the right place to test this, then.